src/main/java/org/sonarsource/sonarlint/ls/util/Utils.java

```diff
@@ -190,4 +193,17 @@ public static TextRangeWithHash textRangeWithHashFromTextRange(TaintVulnerabilit
       textRange.getEndLineOffset(),
       textRange.getHash());
   }
+
+  public static HotspotStatus hotspotStatusOfTitle(String title) {
+    return Arrays.stream(HotspotStatus.values()).filter(hotspotStatus -> hotspotStatus.getTitle().equals(title)).findFirst()
+      .orElseThrow(() -> new IllegalArgumentException("There is no such title of the hotspot status: " + title));
```

Comment:
maybe `there is no such value for hotspot status:` or `there is no such hotspot status:`

Reply:
Addressed in [c411b03](https://github.com/SonarSource/sonarlint-language-server/pull/252/commits/c411b03b7bc01dc3802ed6f65a0d8640a183059c)